x-pack/plugins/upgrade_assistant/server/lib/kibana_status.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DeprecationsServiceStart, IScopedClusterClient } from 'kibana/server';
+import { DomainDeprecationDetails, SavedObjectsClientContract } from 'src/core/server/types';
+
+export const getKibanaUpgradeStatus = async ({
```

Comment:
Should we add some tests for this the same way we have for `es_deprecation_status`?